app/models/customer.rb

```diff
@@ -69,7 +69,7 @@ def applicable_timezone
   end
 
   def applicable_invoice_grace_period
-    return invoice_grace_period if invoice_grace_period.positive?
+    return invoice_grace_period if invoice_grace_period.present?
```

Comment:
we do not need the `positive` test anymore?

Reply:
nope - we just have to check if `invoice_grace_period` is set or not now
previously, we were checking because by default, value was equal to zero